### DISCUSSION TITLE
Developer's Meeting 03-05-2021

### DISCUSSION THREAD

**Discussion by JohnGriffiths** [NeuroTechX/EEG-ExPy#83] [Developer's Meetings]
**Attendees:**

@JohnGriffiths 
@JadinTredup 
@m9h
@oreHGA 
@ErikBjare 
@retiutut 

**Meeting Agenda:**

- Open-ended discussion on the project and project organization and management practices. 
- In particular, soliciting thoughts on this from Ore and Erik

Questions for the group: 
- Within the 'core' group, definition (as far as useful / necessary ) of roles, commitment level
- Assign individuals nominal responsibility to parts of the code
- Key organizational practices we should adopt?

**Meeting Notes:**

MNE-python contributor guidelines - see [here](https://github.com/mne-tools/mne-python/blob/main/CONTRIBUTING.rst) and [here](https://mne.tools/stable/install/contributing.html#building-the-documentation)

Github roadmap - [here](https://github.com/github/roadmap) (also pointed to be Ore in [this thread](https://github.com/NeuroTechX/eeg-notebooks/discussions/64) )

**Comment by ErikBjare** [comment]
Editing as the meeting goes along...

 - Discussed mypy with `check_untyped_defs=false`
 - [ ] Write a CONTRIBUTING.md
 - [x] Set up Issue templates
 - Time commitment

**Reply by oreHGA** [reply]
> [ ] Set up Issue templates

Here's a pretty good example of issue templates we can leverage  https://github.com/microsoft/PowerToys/tree/master/.github/ISSUE_TEMPLATE

We'd need to tweak them a bit to match our scenarios but I like the groupings

**Reply by oreHGA** [reply]
PR enabling issue templates here https://github.com/NeuroTechX/eeg-notebooks/pull/87
